.vitepress/theme/lhm.css

```diff
@@ -72,6 +72,16 @@
   min-width: 169px;
 }
 
+.cb {
+    clear:both;
+}
```

Comment:
What is this for?

Reply:
@FabianWilms https://github.com/it-at-m/opensource.muenchen.de/pull/193/files#diff-c82f085e20eb2a26d68411c24f3ed822e0ca6de2c8cd06ea0ee7f5212f82eeeeR44

Reply:
I saw the usage :D As I haven't checked out your changes I don't understand why you would need to use https://developer.mozilla.org/en-US/docs/Web/CSS/clear#both

Reply:
I  want a clear new paragraph after the right floated image above